source/Octopus.Tentacle/Octopus.Tentacle.csproj

```diff
@@ -52,7 +52,7 @@
 		<PackageReference Include="KubernetesClient" Version="13.0.26" />
 	</ItemGroup>
 	<ItemGroup>
-		<PackageReference Include="Octopus.Client" Version="14.3.1348" />
+		<PackageReference Include="Octopus.Client" Version="14.3.1385-ap-register-environment-slugs" />
```

Comment:
All good once this is updated.

Reply:
Updated in [`ca38892` (#922)](https://github.com/OctopusDeploy/OctopusTentacle/pull/922/commits/ca388929276916a371f63ccf9c54a5b5cc16f62c)